src/script/page/MainContent/MainContent.tsx

```diff
@@ -235,6 +237,7 @@ const MainContent: FC<MainContentProps> = ({
                 userState={userState}
                 isRightSidebarOpen={isRightSidebarOpen}
                 openRightSidebar={openRightSidebar}
+                handleMLSUpdate={onRefresh}
```

Comment:
I'd stick to `onRefresh` name instead of `handleMLSUpdate` for the callback. Conversation component doesn't really need to know what this refresh will trigger.

Reply:
fixed